tests/src/Kernel/Entity/OgRoleTest.php

```diff
@@ -69,9 +86,10 @@ public function testRoleCreate() {
       ->setLabel('Content editor')
       ->setGroupType('entity_test')
       ->setGroupBundle('group')
+      ->setGroupID(1)
       ->save();
 
-    $this->assertEquals('entity_test-group-content_editor', $og_role->id());
+    $this->assertEquals('entity_test-group-1-content_editor', $og_role->id());
 
```

Comment:
OK I get it now. This tests that if a group ID is set, then the generated role ID contains the group ID. The negative case (no group ID set) is tested in the previous test.

Am I right in assuming that a group ID is optional, and is only needed when you are creating two roles of the same type/bundle pair?